README.MD

```diff
@@ -46,6 +46,7 @@ render(
 | `format` | `String` | Degrees format | `format="°C"` | '' |
 | `size` | `String` | Thermometer size. Can be small, normal or large | `size="large"` | normal |
 | `height` | `Number` | Thermometer height | `height={300}` | 200 |
+| `reverseGradient` | `boolean` | Reverse the gradient | `reverseGradient={"true"}` | false |
```

Comment:
Now reverseGradiente expect a string not a boolean value. Change this to:

reverseGradient={true}

Reply:
:+1: